desktop/mode.py

```diff
@@ -50,115 +51,208 @@ def update(self, kind, **info):
+                         "\nSPACE = PREDICT ALL FRAMES"
+                         "\nESC = CANCEL")
+
+        elif self.action == "RELABEL":
+            self.text = ("\nRelabel annotations?"
+                         "\nSPACE = RELABEL IN ALL FRAMES"
+                         "\nP = PRESERVE 3D INFO WHILE RELABELING"
+                         "\nS = RELABEL THIS FRAME ONLY"
+                         "\nU = UNIQUELY RELABEL EACH LABEL"
+                         "\nESC = CANCEL")
+
     def update_prompt(self):
-        text = ""
-        answer = "SPACE = CONFIRM\nESC = CANCEL"
+        self.text = ""
 
         if self.kind == "SELECTED":
-            text = "\nSELECTED {}".format(self.label)
+            self.text = "\nSELECTED {}".format(self.label)
 
         elif self.kind == "MULTIPLE":
-            text = "\nSELECTED {}, {}".format(self.label_1, self.label_2)
+            self.text = "\nSELECTED {}, {}".format(self.label_1, self.label_2)
 
         elif self.kind == "QUESTION":
+
+            self.set_question_single()
+
+            self.set_question_multiframe_options()
+
             if self.action == "SAVE":
-                text = ("\nSave current file?"
-                    "\nSPACE = SAVE"
-                    "\nESC = CANCEL")
+                self.text = ("\nSave current file?"
+                             "\nSPACE = SAVE"
+                             "\nESC = CANCEL")
 
-            elif self.action == "REPLACE":
-                text = ("\nReplace {} with {}?"
-                    "\nSPACE = REPLACE IN ALL FRAMES"
-                     "\nS = REPLACE IN FRAME {} ONLY"
-                     "\nESC = CANCEL").format(self.label_2, self.label_1, self.frame_2)
+            elif self.action == "WATERSHED":
+                self.text = ("\nPerform watershed to split {}?"
+                             "\n{}").format(self.label_1, self.simple_answer)
 
-            elif self.action == "SWAP":
-                if self.frame_1 == self.frame_2:
-                    text = ("\nSwap {} & {}?"
-                        "\nSPACE = SWAP IN ALL FRAMES"
-                        "\nS = SWAP IN FRAME {} ONLY"
-                        "\nESC = CANCEL").format(self.label_2, self.label_1, self.frame_2)
-                else:
-                    text = ("\nSwap {} & {}?"
-                        "\nSPACE = SWAP IN ALL FRAMES"
-                        "\nESC = CANCEL").format(self.label_2, self.label_1)
+        elif self.kind == "PROMPT":
+            self.set_prompt_text()
 
-            elif self.action == "PARENT":
-                text = "\nMake {} a daughter of {}?\n{}".format(self.label_2, self.label_1, answer)
+        elif self.kind == "DRAW":
+            self.text = ("\nUsing conversion brush to replace {} with {}."
+                         "\nUse ESC to stop using the conversion brush.").format(
+                self.conversion_brush_target, self.conversion_brush_value)
```

Comment:
```suggestion
        elif self.kind == "DRAW":
            self.text = ("\nUsing conversion brush to replace {} with {}."
                         "\nUse ESC to stop using the conversion brush.").format(
                             self.conversion_brush_target, self.conversion_brush_value)
```

Reply:
I'm not sure about this one. The multiple strings definitely fall under "need to be aligned with opening delimiter" but I think the `.format` hanging indent applies to the indentation level of `self.text`, not the string. I think we can leave it as is and make that modification later, once we start linting with pytest, if we need.

Reply:
OK no worries